### ISSUE TITLE
Wrong remote IP detection

### ISSUE THREAD

**Issue by gjedeer** [OtherOfIce/VPN-Jail#1]
vpnJail.sh could not connect to the VPN IP, I figured it was because of wrong remote IP detection.

```
grep "^[^#;]" ch7.nordvpn.com.tcp443.ovpn | grep remote | cut -d " " -f2
185.183.104.58
remote-random
server
```

I suggest filtering out lines containing `remote-cert`, `remote-random`, `persist-remote-ip`

**Comment by OtherOfIce** [comment]
Fixed, sorry about being late had some life things come up :smile:

**Comment by gjedeer** [comment]
Thanks!

On March 10, 2017 2:29:37 PM CET, Holbrad <notifications@github.com> wrote:
>Closed #1.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/OtherOfIce/VPN-Jail/issues/1#event-994883711

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
